packages/x-adapter-platform/src/types/platform-adapter.types.ts

```diff
@@ -6,15 +6,12 @@ import {
   PopularSearchesResponse,
   QuerySuggestionsRequest,
   QuerySuggestionsResponse,
-  NextQueriesRequest
+  NextQueriesRequest,
+  SearchRequest,
+  SearchResponse
 } from '@empathyco/x-types';
-import { BaseRequest, SearchRequest, TaggingRequest } from './request.types';
-import {
-  NextQueriesResponse,
-  RelatedTagsResponse,
-  SearchResponse,
-  TopClickedResponse
-} from './response.types';
+import { BaseRequest, TaggingRequest } from './request.types';
+import { NextQueriesResponse, RelatedTagsResponse, TopClickedResponse } from './response.types';
```

Comment:
NextQueriesResponse  and TopClickedResponse should not be imported from here, right?

Reply:
I think that this change is made on the abraham's PR

Reply:
Yes, you're right, it should not be imported from there and that change is made in Abraham's PR.